extensions/amp-accordion/1.0/amp-accordion.js

```diff
@@ -47,6 +47,10 @@ const CONTENT_SHIM_PROP = '__AMP_C_SHIM';
 const SECTION_POST_RENDER = '__AMP_PR';
 const EXPAND_STATE_SHIM_PROP = '__AMP_EXPAND_STATE_SHIM';
 
+const isDisplayLockingEnabledForAccordion = (win) =>
+  isExperimentOn(win, 'amp-accordion-display-locking') &&
+  win.document.body.onbeforematch !== undefined;
```

Comment:
It shouldn't be necessary to check `onbeforematch` here since the Preact layer checks for `content-visibility: hidden-matchable`

Reply:
I agree it's not necessary.  I just figured I'd include it to match with `0.1`.  One benefit of having though is that in the case that the experiment is on, but the `onbeforematch` is `undefined`, we won't be passing in the prop `experimentDisplayLocking` as `true` to the bento component.  Agreed that the bento component still does the right thing since it will still check for `content-visibility: hidden-matchable`, but I figured it wouldn't hurt to check on amp side as well.

Open to updating this one though, what do you think?

Reply:
I'm afraid we have to keep it for now b/c we're duplicating CSS at this time. Once we remove CSS duplication, this can go as well, but by then we're likely to have launched the experiment anyway.

Reply:
Sounds good